client/.env

```diff
@@ -1,2 +1,14 @@
 REACT_APP_SERVER_HOST=localhost
-REACT_APP_SERVER_PORT=2000
\ No newline at end of file
+REACT_APP_SERVER_PORT=2000
+REACT_APP_LOCIZE_PROJECTID = 1c6e09c8-3fe2-47e4-a574-72f3da05efc1
+REACT_APP_LOCIZE_APIKEY = 8a6a92cc-f868-4478-ba57-1426945deed3
+REACT_APP_LOCIZE_REFLNG = en
+REACT_APP_LOCIZE_VERSION = latest
+
+
+
+
+# REACT_APP_LOCIZE_PROJECTID = b660bfc5-52a5-47a8-ace9-37cb21c65d91
+# REACT_APP_LOCIZE_APIKEY = 68b4a710-6acb-4fc4-9f9a-396c47f3d102
+# REACT_APP_LOCIZE_REFLNG = en
+# REACT_APP_LOCIZE_VERSION = latest
```

Comment:
Do we need these comments?

Also, add an empty line at the EOF.

Reply:
Cleaned.